### ISSUE TITLE
Add mojom support (e.g., for syntax highlighting) to github

### ISSUE THREAD

**Issue by viettrungluu-cr** [domokit/mojo#693]
I think this is the thing to look at: https://github.com/github/linguist

Then .mojom files will be syntax highlighted. And we'll be able to have mojom code blobs (e.g., in markdown) that's correctly highlighted.

**Comment by viettrungluu-cr** [comment]
@vardhan-cr -- maybe something to do in your copious (!) spare time?

**Comment by vardhan-cr** [comment]
I don't think we can add a new/custom language to github without sending them a pull request to Linguist, and I don't think mojom qualifies yet:

"We try only to add languages once they have some usage on GitHub. In most cases we prefer that each new file extension be in use in hundreds of repositories before supporting them in Linguist." 

https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language
